README.md

```diff
@@ -40,7 +38,318 @@ Under construction.
 <img src="https://img.shields.io/github/last-commit/memphisdev/memphis-broker?color=61dfc6&label=last%20commit">
 </p>
 
-**[Memphis{dev}](https://memphis.dev)** is an open-source real-time data processing platform<br>
-that provides end-to-end support for in-app streaming use cases using Memphis distributed message broker.<br>
-Memphis' platform requires zero ops, enables rapid development, extreme cost reduction, <br>
-eliminates coding barriers, and saves a great amount of dev time for data-oriented developers and data engineers.
+**[Memphis](https://memphis.dev)** is a next-generation alternative to traditional message brokers.<br><br>
+A simple, robust, and durable cloud-native message broker wrapped with<br>
+an entire ecosystem that enables cost-effective, fast, and reliable development of modern queue-based use cases.<br><br>
+Memphis enables the building of modern queue-based applications that require<br>
+large volumes of streamed and enriched data, modern protocols, zero ops, rapid development,<br>
+extreme cost reduction, and a significantly lower amount of dev time for data-oriented developers and data engineers.
+
+## Installation
+
+```sh
+$ pip3 install memphis-py
+```
+
+## Importing
+
+```java
+import com.memphis.Headers;
+import com.memphis.Memphis;
+```
+
+### Connecting to Memphis
+
+First, we need to create Memphis `object` and then connect with Memphis by using `memphis.connect`.
+
+```java
+public static void main(String[] args) {
+    try {
+        Memphis memphis = new Memphis();
+        memphis.connect(
+            "<memphis-host>",
+            "<application-type username>",
+            "<broker-token>",
+            "<port>", // defaults to 6666
+            true, // reconnect, defaults to true
+            3, // max_reconnect, defaults to 3
+            1500, // reconnect_interval_ms, defaults to 1500
+            1500, // timeout_ms, defaults to 1500
+            "<key-client.pem>", // key_file, for TLS connection
+            "<cert-client.pem>", // cert_file, for TLS connection
+            "<rootCA.pem>" // ca_file, for TLS connection
+        ).get(); // wait for the connection to complete
+
+        // do other things with the memphis object here
+
+        memphis.close().get(); // wait for the connection to close
+    } catch (Exception e) {
+        e.printStackTrace();
+    }
+}
+```
+
+Once connected, the entire functionalities offered by Memphis are available.
+
+### Disconnecting from Memphis
+
+To disconnect from Memphis, call `close()` on the memphis object.
+
+```java
+memphis.close().get();
+```
+
+### Creating a Station
+
+_If a station already exists nothing happens, the new configuration will not be applied_
+
+```java
+Station station = memphis.createStation(
```

Comment:
.get() is needed I guess

Reply:
in the python README the create station is without await so I guess is not needed